source/MaterialXView/Editor.cpp

```diff
@@ -458,6 +458,7 @@ void PropertyEditor::addItemToForm(const mx::UIPropertyItem& item, const std::st
             }
         });
         v1->setSpinnable(editable);
+        v1->setEditable(editable);
```

Comment:
This looks like a merge mistake?

Reply:
I don't think so, it's for v1 here and v2 below.

Setting it to editable makes it possible to edit the value by typing in a number, in addition to spinning the up and down arrow on the control.